types/webxr/index.d.ts

```diff
@@ -1242,6 +1243,11 @@ interface XRWebGLBinding {
     getDepthInformation(view: XRView): XRWebGLDepthInformation | null | undefined;
 }
 
+enum XRInteractionMode {
+    ScreenSpace = "screen-space",
+    WorldSpace = "world-space"
+}
+
```

Comment:
```suggestion
type XRInteractionMode = "screen-space" | "world-space";
```

We don't use enums but types. there is no XRInteractionMode namespace, and it has no ScreenSpace nor WorldSpace members.